pynitrokey/nk3/admin_app.py

```diff
@@ -178,8 +180,15 @@ def set_config(self, key: str, value: str) -> None:
 
     def factory_reset(self) -> None:
         try:
+            local_print(
+                "Please touch the device to confirm the operation", file=sys.stderr
+            )
             reply = self._call(AdminCommand.FACTORY_RESET, response_len=1)
-            assert reply
+            if reply is None:
+                local_critical(
+                    "Factory reset is not supported by the device", support_hint=False
```

Comment:
I would say “in this firmware version” instead so that it is clear why it is not supported.